docs/first-steps.rst

```diff
@@ -0,0 +1,67 @@
+.. _first_steps:
+
+===========
+First steps
+===========
+
+The purpose of this guide is to illustrate the possible uses of geomstats.
+
+**Install geomstats.**
```

Comment:
Nitpicking, but I'd use title case for all the section headers and not use a period at the end.